app/controllers/user.js

```diff
@@ -89,23 +94,29 @@ exports.signIn = (req, res, next) => {
 };
 
 exports.signUp = (req, res, next) => {
-  if (!emailValid(req.body.email)) {
-    logger.error(`Email: ${req.body.email} invalid.`);
-    return next(errors.emailNotValid(req.body.email));
-  }
-  if (!passwordValid(req.body.password)) {
-    logger.error('Password invalid.');
-    return next(errors.passwordInvalid);
-  }
-
   const user = {
     firstName: req.body.firstName,
     lastName: req.body.lastName,
     email: req.body.email,
     password: req.body.password,
     admin: false
   };
-
+  generateUser(user, next)
+    .then(auxUser => {
```

Comment:
same as before this `auxUser` is the actual user, so if you name it user then you can also user `.send({ user })` and it will look better and also be more descriptive